roles/kubespray-defaults/defaults/main/download.yml

```diff
@@ -275,7 +275,7 @@ haproxy_image_tag: 2.8.2-alpine
 # Coredns version should be supported by corefile-migration (or at least work with)
 # bundle with kubeadm; if not 'basic' upgrade can sometimes fail
 
-coredns_version: "{{ 'v1.10.1' if (kube_version is version('v1.27.0', '>=')) else 'v1.9.3' }}"
+coredns_version: "{{ 'v1.11.1' if (kube_version is version('v1.27.0', '>=')) else 'v1.9.3' }}"
```

Comment:
```suggestion
coredns_version: "{{ 'v1.11.1' if (kube_version is version('v1.29.0', '>=')) else 'v1.10.1' }}"
```

Reply:
Hello @batazor 

Please rebase and check my suggestion above. Also as per this comment https://github.com/kubernetes-sigs/kubespray/pull/10719#issuecomment-1861975302 coredns v1.11.1 supports kubernetes 1.29.